cmd/boots/main.go

```diff
@@ -220,15 +225,17 @@ func main() {
 	}
 }
 
-// getWorkflowFinder returns a no-op workflow finder if tinkerbell is not the backend
-func getWorkflowFinder() (client.WorkflowFinder, error) {
+// getWorkflowFinder returns a no-op workflow finder if tinkerbell or kubernetes is not the backend
```

Comment:
hmm I think standalone should be allowed too right @jacobweinstock ?

Reply:
ah yes. seems like standalone should be in there.

Reply:
How does that apply? Standalone doesn't have a Tinkerbell backend right?

Reply:
standalone only applies to the hardware fetcher, workflows can still happen with standalone.

Reply:
With which Tinkerbell data model? Tink on K8s or Tink on postgres?

Reply:
For backwards compatibility, I'll use Tink on Postgres. We'll have to figure how to transition this later

Reply:
I wonder if standalone should also imply workflows from json too :thinking:

Reply:
@jacobweinstock I take it you use standalone w/ workflows, thoughts?

Reply:
i use standalone mainly for testing, this seems fine regardless.